otherlibs/unix/unix.ml

```diff
@@ -597,6 +597,7 @@ type socket_bool_option =
     SO_DEBUG
   | SO_BROADCAST
   | SO_REUSEADDR
+  | SO_REUSEPORT
```

Comment:
I know that no one _should_ rely on the order of constructors, but shouldn’t this have been added at the end of the type? (and given that we haven’t frozen 4.12, should we move it)

Given the ease with which the wrong stubs can be loaded on bytecode, it doesn’t seem great to have versions of the C stubs which interpret the same numeric constant differently?

Reply:
It could be prudent to move the new constructor last.  However, many other things can go wrong if the wrong stub library is dynamically loaded, so this could just as well make no difference in safety.